README.md

```diff
@@ -21,10 +21,14 @@ description: Get device information.
 #         under the License.
 -->
 
-# cordova-plugin-device
+# cordova-plugin-device (with zebra s/n support on Android 10+)
```

Comment:
This plugin, the generated npm package and this repository should be renamed to `mobisys-cordova-plugin-device`. Also, there's no need to mention the feature in the title - the title should only contain the project name.

```suggestion
# mobisys-cordova-plugin-device
```